src/scripts/hero-animation/hero.js

```diff
@@ -0,0 +1,210 @@
+(function() {
```

Comment:
I wonder if we should convert this to an ES6-style module and then import it from `main.js`. That way it ends up in the `bundle.js`. Then again, this script is only used on the homepage. Probably worth discussing with the rest of the team and deciding an overall approach for this JS on the website.